### DISCUSSION TITLE
How does vorta handle missed timewindows for schedules?

### DISCUSSION THREAD

**Discussion by newhinton** [borgbase/vorta#998] [FAQ]
I am often not using my laptop when i am home, and therefore there is no predictable time when an backup should be possible. It is rather easy to miss those timeframes for a prolonged time. How does Vorta handle this case?

this may contribute to https://github.com/borgbase/vorta/discussions/992, and maybe the daily-schedule needs an option to handle such cases like a "when missed, do as soon as possible"-option if it doesn't already behave this way

**Comment by m3nu** [comment]
In progress in #908
